qml/js/yleApi.js

```diff
@@ -126,6 +126,14 @@ var api = (function(appId, appKey) {
              .then(mapPrograms);
      }
 
+     function getProgramsBySeriesId(seriesId, limit, offset) {
+         var url = apiUrl + programsUrl + '&series=' + seriesId + '&availability=ondemand&mediaobject=video&offset=' + offset + '&limit=' + limit;

```

Comment:
do we need the contentprotection for this also: `&contentprotection=22-0,22-1&availability=ondemand` ?

Reply:
Yes, it should be there, I will move those common parameter to the `programsUrl` variable.